V2DriverDemo.md

```diff
@@ -0,0 +1,352 @@
+# Azure Disk CSI Driver V2
+
+The Azure Disk CSI Driver V2 enhances the Azure Disk CSI Driver to improve scalability and reduce pod failover latency. It uses shared disks to provision attachment replicas on multiple cluster nodes and integrates with the pod scheduler to ensure a node with a attachment replica is chosen on pod failover. It is beneficial for both single zone use case as well as multi zone use case that uses [Zone Redundant Disks](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-redundancy#zone-redundant-storage-for-managed-disks). This demo is based off [this guide](https://github.com/mohmdnofal/aks-best-practices/blob/master/stateful_workloads/zrs/README.md)
+
+# Azure Disk CSI Driver V2 with ZRS Demo Introduction 
+In this demo we will create a 3 nodes cluster distributed across 3 availability zones, we will deploy a single mysql pod, ingest some data there, and then drain the node hosing the pod, triggering a failover. This means that we took one node offline, triggering the scheduler to migrate your pod to another node. 
+
+# Demo
+
+1. Create the cluster 
+```shell 
+
+#Set the parameters
+LOCATION=northeurope # Location 
+AKS_NAME=az-zrs
+RG=$AKS_NAME-$LOCATION
+AKS_CLUSTER_NAME=$AKS_NAME-cluster # name of the cluster
+K8S_VERSION=$(az aks get-versions  -l $LOCATION --query 'orchestrators[-1].orchestratorVersion' -o tsv)
+
+
+##Create RG
+az group create --name $RG --location $LOCATION
+
+
+## create the cluster 
+az aks create \
+-g $RG \
+-n $AKS_CLUSTER_NAME \
+-l $LOCATION \
+--kubernetes-version $K8S_VERSION \
+--zones 1 2 3 \
+--generate-ssh-keys 
+
+
+## get the credentials 
+
+az aks get-credentials -n $AKS_CLUSTER_NAME -g $RG
+
+## test
+
+kubectl get nodes  
+
+NAME                                STATUS   ROLES   AGE   VERSION
+aks-nodepool1-20996793-vmss000000   Ready    agent   77s   v1.21.2
+aks-nodepool1-20996793-vmss000001   Ready    agent   72s   v1.21.2
+aks-nodepool1-20996793-vmss000002   Ready    agent   79s   v1.21.2
+```
+
+2. Install the Azure Disk CSI Driver v2
+
+As of K8s 1.21 CSI became the default in storage drivers, same in AKS, you can see the default storage class now pointing to v1 azure disk CSI driver. To demonstrate the Azure Disk CSI Driver V2, first we will install it such that it runs side by side with the V1 driver. Tips for troubleshooting helm installation [here](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/668a54a797fd90f015ce2b89ff2fbac2d0a4600b/charts/README.md).
```

Comment:
I find this first sentence difficult to parse. Perhaps, break up the Kubernetes and AKS descriptions into two separate sentences? Also, "Azure Disk CSI Driver" should always use Proper Name Casing.